nodes/matrix/change_matrix_pivot.py

```diff
@@ -0,0 +1,77 @@
+import bpy
+from bpy.props import *
+from ... tree_info import keepNodeState
+from ... events import executionCodeChanged
+from ... algorithms.rotation import generateDirectionToRotationCode
+from ... base_types.node import AnimationNode
+
+pivotTypeItems = [
+    ("MATRIX", "Pivot Matrix", "Change Pivot by Matrix, like parent", "", 0),
```

Comment:
It's not the only place like this. guess. .. 
What is the diff to, say, vector math? 
Is there a problem with that comma in the text?